lineapy/api/api.py

```diff
@@ -211,16 +212,28 @@ def _try_write_to_db(value: object) -> Path:
                 k=7,
             )
         )
+        + ".pkl"
     )
+
+    artifact_storage_dir = UPath(options.safe_get("artifact_storage_dir"))
+    if not artifact_storage_dir.exists():
+        artifact_storage_dir.mkdir(parents=True)
```

Comment:
do we need to pass an exists=True here?

Reply:
We don't need to even check whether the folder exists at all. `options.safe_get("artifact_storage_dir")` will create the folder if it doesn't exist.

Reply:
in that case, remove the if statement?

Reply:
done